Index.html

```diff
@@ -186,5 +186,253 @@ <h2 class="hero-text-title">I'm a Full Stack Web Developer</h2>
       </div>
 
     </section>
+
+    <section class="recent-works" id="recent-works-id">
+
+      <h2 class="recent-works-title"><span>My Recent Works</span></h2>
+
+      <div class="main-work-box">
+        <div class="main-work-image"></div>
+        <div class="main-work-text-box">
+          <h3 class="main-work-text-title">Lifestyle-Articles Web Application</h3>
+          <p class="main-work-text-area">
+          Based on the website design of <a href="https://www.behance.net/gallery/14554909/liFEsTlye-Mobile-version" class="nelson-sakwa">Nelson Sakwa on Behance</a>. Build with Ruby on Rails, the project has the models, views and controllers of each of the resources User, Article, Vote and Category. The User has authentication and can perform any of the CRUD operations on the Articles and can create and delete a Vote. Article and Category has a many to many relationship. The Article with the most Vote will be displayed in the hero section of the main page.
+          </p>
+          <div class="main-work-text-skills-box">
+            <span class="skill">HTML</span>
+            <span class="skill">CSS</span>
+            <span class="skill">Ruby</span>
+            <span class="skill">Rails</span>
```

Comment:
- [ ] Please wrap the project technologies in `<ul>` and `<li>` to follow the project requirements. When we have similar elements on the page it is good practice to wrap them in a list and it will also make sure the code is semantic.

### See the image of the requirement 

![image](https://user-images.githubusercontent.com/83803414/166122553-f0afdfb3-a1fe-42d2-b745-39828d630d84.png)